cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/UserApi.java

```diff
@@ -132,24 +133,32 @@
     Call<Void> requestVerification();
 
     /**
-     * @deprecated look for {@link #savePaymentMethod(PaymentMethodDTO)}
+     * @deprecated look for {@link #savePaymentMethod(StripePaymentMethodDTO)}
      */
     @Deprecated
     @POST( "user/payment/card" )
     Call<Void> saveCardData( @Body CardDataDTO data );
 
     /**
-     * @deprecated look for {@link #saveTokenData(TokenDataDTO)}
+     * @deprecated look for {@link #saveTokenData(StripeTokenDTO)}
      */
     @Deprecated
     @POST( "user/payment/sepa" )
     Call<Void> saveSepaData( @Body SepaDataDTO data );
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)